src/wallet/bdb.cpp

```diff
@@ -791,6 +791,7 @@ void BerkeleyDatabase::AddRef()
     } else {
         m_refcount++;
     }
+    assert(m_refcount <= 1);
```

Comment:
this is failing for me:
`AssertionError: Unexpected stderr Assertion failed: (m_refcount <= 1), function AddRef, file wallet/bdb.cpp, line 794. !=`